docs/components/modeler/web-modeler/process-application-pipeline.md

```diff
@@ -0,0 +1,92 @@
+---
+id: process-application-pipeline
+title: Process application development pipeline
+description: The Web Modeler provides a lightweight development pipeline you can use to quickly develop and progress low-risk process application releases through the stages of a standard development life cycle.
+---
+
+import DeployProcessApplicationDiagramImg from './img/process-applications/diagram-process-application-pipeline.png'
+import DeployStagesImg from './img/process-applications/define-stages.png'
+
+You can use the Web Modeler development pipeline to quickly develop and progress low-risk process application releases through the stages of a standard development life cycle.
+
+<p><img src={DeployProcessApplicationDiagramImg} alt="Process application file list" /></p>
+
+:::caution
+For business-critical and higher-risk processes that require strict governance and/or quality requirements, you can [integrate Web Modeler into your CI/CD pipelines](/docs/guides/devops-lifecycle/integrate-web-modeler-in-ci-cd.md).
+:::
+
+## Model
+
+During the modeling stage, you will typically:
+
+- [Create a process application](create-a-process-application.md) and select a default development cluster to deploy to.
+- Invite other users to collaborate on the process application.
+- Define and setup the clusters and deployment stages that you will use in your development pipeline.
+- Model your diagrams and any associated resources, and fix any errors shown in the modeler.
+
+### Deployment stages
+
+You can deploy and promote an application process version between the stages of a standard development life cycle (development, testing, staging, production).
```

Comment:
the deployment stages are on the deployment pipeline - the dev lifecycle is much bigger than this

Reply:
Edited: You can deploy and promote an application process version through the stages of a standard deployment pipeline (development, testing, staging, and production).